CromIAM/src/main/scala/cromiam/sam/SamClient.scala

```diff
@@ -98,15 +114,16 @@ class SamClient(scheme: String, interface: String, port: Int, log: LoggingAdapte
   private lazy val samBaseUri = s"$scheme://$interface:$port"
   private lazy val samBaseResourceUri = s"$samBaseUri/api/resource"
   private lazy val samBaseCollectionUri = s"$samBaseResourceUri/workflow-collection"
+  private lazy val samSubmitWhitelistUri = s"$samBaseResourceUri/caas/submit/action/get_whitelist"
 
 }
 
 object SamClient {
   case object SamDenialException extends Exception("Access Denied")
-  val SamDenialResponse = HttpResponse(status = StatusCodes.Unauthorized, entity = SamDenialException.getMessage)
+
+  val SamDenialResponse = HttpResponse(status = StatusCodes.Forbidden, entity = SamDenialException.getMessage)
```

Comment:
ToL: Oops, that comment above should probably have been here: we should make sure we don't leak anything from SAM in the exception message that the bad user didn't already tell us.